### DISCUSSION TITLE
FIXED my code is off by a little and can't find the problem

### DISCUSSION THREAD

**Discussion by tutucea** [skyfielders/python-skyfield#886] [Q&A]
took the code from here https://rhodesmill.org/skyfield/ .. its the first one as you scroll down

python umf.py 
11h 17m 07.65s
+05deg 31' 51.3"
2.41102 au
now i converted this to Astrological sign, degrees, minutes and seconds
 
python a.py
Astrological Position of Mars: Virgo 19 degrees 14 arcminutes 59.19176398942682 arcseconds

and this is the correct position from astro.com( i doubles checked with other websites and this is correct )
Mars 18 16'12" 5n25 in Virgo ... Can you guys help me figure it out .. i can post the actual code for it

**Comment by Bernmeister** [comment]
Yes, post a full and functioning script which demonstrates the error...

**Reply by Bernmeister** [reply]
If you enclose the script with triple backchecks you'll get

```
nicely formatted code
```

(and the point of the full functioning script is that anyone who wants to jump in to help you can copy the script and run it, without having to muck around (make it as easy as possible for someone to help you).

**Comment by tutucea** [comment]


**Reply by tutucea** [reply]
```Current Planet Positions:
UTC Time: 2023-08-22 20:29:12.068913
Sun Astrological Sign: Leo 29° 29' 48.35"
Moon Astrological Sign: Scorpio 10° 57' 14.30"
Mercury Astrological Sign: Virgo 21° 48' 27.01"
Venus Astrological Sign: Leo 15° 8' 21.95"
Mars Astrological Sign: Virgo 26° 59' 43.96"
Jupiter Astrological Sign: Taurus 15° 18' 53.43"
Saturn Astrological Sign: Pisces 4° 10' 29.31"
Uranus Astrological Sign: Taurus 23° 3' 32.21"
Neptune Astrological Sign: Pisces 26° 59' 51.30"
Pluto Astrological Sign: Capricorn 28° 25' 0.85"
```
okay progress has been made and everything works just fine .  now the last piece of the puzzle is figuring out the NODES
